models/quality_measures/quality_measures_models.yml

```diff
@@ -131,6 +131,57 @@ models:
       - name: tuva_last_run
         description: The date and timestamp of the dbt run.
 
+## Shared Exclusions
+  - name: quality_measures__shared_exclusions_frailty
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}{{var('tuva_schema_prefix')}}_quality_measures{% else %}quality_measures{%- endif -%}
+      alias: shared_exclusions_frailty
```

Comment:
Hey @deepsonshrestha, the CI Testing has passed, and I tested on some customer data. Everything is looking good, but I do have one more request. Sorry, I was not specific enough before. Can we change the SQL file names and aliases for all shared models to match the pattern we set for intermediate models?

For example, this model would be named:
`name: quality_measures__int_shared_exclusions_frailty`
`alias: _int_shared_exclusions_frailty`

This helps control how the tables are displayed in the database. We like to ensure intermediate models are grouped, and final models are displayed at the top (or bottom, depending on the IDE).

Reply:
No worries @sarah-tuva. I have refactored the codebase. Please have a look. Thanks!